README.md

```diff
@@ -138,7 +123,118 @@ This temporary directory is then added to the `source_paths` of the Rails
 generator system, allowing all of its ERb templates and files to be referenced
 when the application template script is evaluated.
 
-Rails generators are very lightly documented; what you’ll find is that most of
-the heavy lifting is done by [Thor](http://whatisthor.com/). The most common methods used by this
-template are Thor’s `copy_file`, `template`, and `gsub_file`. You can dig into
-the well-organized and well-documented [Thor source code](https://github.com/erikhuda/thor) to learn more.
+Rails generators are very lightly documented; what you’ll find is that most of the heavy lifting is done by  [Thor](http://whatisthor.com/). Thor is a tool that allows you to easily perform command line utilities.  The most common methods used by this template are Thor’s `copy_file`, `template`, and `gsub_file`. You can dig into the well-organized and well-documented [Thor source code](https://github.com/erikhuda/thor) to learn more.
+If any file finishes with `.tt`, Thor considers it to be a template and places it in the destination without the extension `.tt`. 
+
+## Tooling choices and configuration
+  
+#### Testing: [webdrivers](https://github.com/titusfortner/webdrivers)
+auto-installs headless Chrome
+
+#### ENV managment: [dotenv](https://github.com/bkeepers/dotenv)  
+in place of the Rails `secrets.yml`
+  
+#### Application web server: [puma](https://github.com/puma/puma)
+  
+#### Security Auditing: [brakeman](https://github.com/presidentbeef/brakeman) and [bundler-audit](https://github.com/rubysec/bundler-audit)
+
+#### Editor code style settings: [EditorConfig](https://editorconfig.org/) 
+“EditorConfig helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs. The EditorConfig project consists of a file format for defining coding styles and a collection of text editor plugins that enable editors to read the file format and adhere to defined styles. EditorConfig files are easily readable and they work nicely with version control systems” 
+
+See the `editorconfig` file for Ackama’s styles
+
+#### Git hook manager: [Overcommit](https://github.com/sds/overcommit) 
+Git has a way to fire off custom scripts when certain important actions occur, by using [Git hooks ](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)
+
+Overcommit is a tool to manage and configure Git hooks
+
+Edit `overcommit.yml` to configure the Overcommit hooks you wish to use
+
+#### Code coverage measurement: [Simplecov](https://github.com/simplecov-ruby/simplecov)
+Code coverage is a measurement of how many lines/blocks/arcs of your code are executed while the automated tests are running.
+
+SimpleCov is a code coverage analysis tool for Ruby. It provides an API to filter, group, merge, format, and display the results
+
+#### JS package manager: [Yarn](https://yarnpkg.com/)
```

Comment:
```suggestion
#### JS package manager: [Yarn](https://classic.yarnpkg.com/lang/en/)
```

We're not using Yarn 2 - it's probably worth noting that explicitly too, but that might make things a bit more confusing.

Generally a standard yarn install (i.e `npm i -g yarn`) should always give you Yarn 1.